### PR TITLE
[enterprise-3.10] Add expanded example for curator

### PR DESCRIPTION
**Pull request by jboxman**
- Based on https://github.com/openshift/openshift-docs/pull/13758
- BZ https://bugzilla.redhat.com/show_bug.cgi?id=1633337

### CODE REVIEW

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -1653,34 +1653,50 @@ Use:
...
         days: 30
 ----
 
+<1> Optionally, change the default number of days between run and the run hour and run minute. 
+<2> Delete indices in the *myapp-dev* project older than `1 day`
+<3> Delete indices in the *myapp-qe* project older than `1 week`
+<4> Delete *operations* logs older than `8 weeks`
+<5> Delete all other projects indices after they are `31 days` old
+<6> Delete indices older than 1 day that are matched by the '^project\..+\-dev.*$' regex
+<7> Delete indices older than 2 days that are matched by the '^project\..+\-test.*$' regex
```
Comment (anpingli) [inline install_config/aggregate_logging.adoc]: It is days: 30 above.

Comment (jboxman) [reply install_config/aggregate_logging.adoc]: @anpingli, for item 5? The `days` parameter is `31`, so I'm confused. Thanks.

```
  .defaults: <5>
    delete:
      days: 31
    runhour: 0
    runminute: 0
    timezone: America/New_York
```

**Comment by jboxman** [comment]
Got QE approval in the bug, merging.

**Pull request merged**